### PR TITLE
Matcher to verify allow value for attribute

### PR DESCRIPTION
**Pull request by vyper**
# Usage:
## Class

``` ruby
class Person
  include Lotus::Validations

  attribute :email, format: /@/
end
```
## Spec

``` ruby
it { is_expected.to allow_value("leo@nospam.org").for(:email) }
it { is_expected.to_not allow_value('leo-at-nospam.org').for(:email) }
```

### CODE REVIEW

**Comment by vyper** [comment]
@adilsoncarvalho @marcocarvalho :shipit:

File: spec/shoulda/matchers/validate_presence_matcher_spec.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+require 'spec_helper'
```
Comment (adilsoncarvalho) [inline spec/shoulda/matchers/validate_presence_matcher_spec.rb]: As this PR is all about `allow_value` maybe we could delete this file

**Comment by adilsoncarvalho** [comment]
Pretty neat! :+1:

**Comment by marcocarvalho** [comment]
:+1:  massa... :)

**Pull request merged**